dags/openshift_nightlies/tasks/utils/final_dag_status.py

```diff
@@ -23,7 +23,6 @@ def final_status(**kwargs):
 def get_task(dag):
     return PythonOperator(
         task_id='final_status',
-        provide_context=True,
```

Comment:
This argument is deprecated, removing it.

Reply:
@rsevilla87 this is required as we need to pass airflow context variables to kwargs of final_status.

Reply:
> @rsevilla87 this is required as we need to pass airflow context variables to kwargs of final_status.

https://airflow.apache.org/docs/apache-airflow/2.3.1/release_notes.html?highlight=provide_context#airflow-operators-python-pythonoperator

Reply:
ah okay!